lib/project_types/script/script_project.rb

```diff
@@ -13,6 +14,14 @@ def initialize(dir)
       @language = 'ts'
     end
 
+    def source_file
+      "#{SOURCE_DIR}/#{file_name}"
+    end
+
+    def file_name
+      "script.#{language}"
+    end
```

Comment:
These methods are temporary. This is migrated from the ScriptRepository but this stuff should be determined by the type of project (ie an AssemblyScript project) and that change isn’t in this PR yet. There are a few other changes here like that, but the fix will come in my following refactor PRs. I just wanted to break all my changes down so this PR isn’t huge